docs/specifications/README.md

```diff
@@ -3,9 +3,12 @@
 
 ### Purpose of this directory
 
-This contents of this directory includes detailed specifications for all the
-DNS tests validated by the "zonemaster-engine".
+This contents of this directory includes detailed technical specifications for all the
+DNS tests performed by Zonemaster.
 
-All the test implementation is based on these specifications. The documentation
-in this directory will be useful for developers, DNS experts etc.
+The [tests](tests/README.md) directory contains the test cases that the implementation of Zonemaster
+Engine is based on. The documentation in this directory will be useful for developers, 
+DNS experts etc.
 
+The [methods](methods/README.md) directory contains other specification, currently only specification for
```

Comment:
We should apply better naming here. The name "method" has already been given to [another concept](https://github.com/dotse/zonemaster/blob/master/docs/specifications/tests/Methods.md). I suggest calling the directory "test-types" instead, and maybe rename the "tests" directory to "test-cases".

Reply:
I change to "test-types". Changing from "tests" to "test-cases" affects many files, so I'll do that in a different change.